src/views/Farms/components/FarmTable/Earned.tsx

```diff
@@ -15,9 +15,13 @@ const Amount = styled.span<{ earned: number }>`
 
 const Earned: React.FunctionComponent<EarnedProps> = ({ earnings }) => {
   const { account } = useWeb3React()
-  const displayBalance = earnings !== null && account ? earnings.toLocaleString() : '?'
+  let amountEarned = earnings
+  if (account == null) {
```

Comment:
`!account` is preferable

Reply:
I'm surprised eslint doesn't throw an error for `==`

Reply:
done

Reply:
You can use a ternary with a const. As a rule of thumb, avoiding let is always a good idea.

Reply:
Fixed